app/components/ak-modal/template.hbs

```diff
@@ -5,12 +5,12 @@
   ...attributes
 >
   {{#if @showHeader}}
-    <div local-class='modal-header'>
+    <div local-class='modal-header' data-test-modal='header'>
```

Comment:
ember test selectors will remove all the attr which has `data-test-*`, right? 
In earlier PR it was like data-test='' which will not be excluded from the prod build but this one should do that. Do we have any other reason than this to avoid using `data-test-*` pattern? @Praseetha-KR

Reply:
In the [ember-test-selectors usage guide](https://github.com/simplabs/ember-test-selectors#usage), the examples provided for `data-test-*` attributes are flat unless there is a unique id. I feel it is better to keep it consistent in that way across irene rather than mixing with our own format.

Do you see any advantage of using `data-test-modal='header'` over `data-test-modal-header`?

Reply:
I don't see very drastic advantage over this. But, this could be helpful when we query group of elements like data-test-item='1', data-test-item='2', etc., yeah we can use * for pattern matching but it has more specified.
Also, I saw couple of other packages, most of them using attr with value so, I just chosen this approach.

Reply:
Let's stick to the documented way. Otherwise we'll have to explain to a new contributor why it was done different than the recommended way.